src/main/java/seedu/pill/command/HelpCommand.java

```diff
@@ -0,0 +1,255 @@
+package seedu.pill.command;
+
+import seedu.pill.exceptions.PillException;
+import seedu.pill.util.ItemMap;
+import seedu.pill.util.PillLogger;
+import seedu.pill.util.Storage;
+import seedu.pill.util.StringMatcher;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.logging.Logger;
+
+/**
+ * Represents a command that displays help information about available commands.
+ */
+public class HelpCommand extends Command {
+    private static final Logger logger = PillLogger.getLogger();
+    private static final List<String> VALID_COMMANDS = Arrays.asList(
+            "help", "add", "delete", "edit",
+            "list", "exit", "stock-check");
+    private final String commandName;
+    private final boolean verbose;
+
+    public HelpCommand(String commandInput, boolean verbose) {
```

Comment:
Maybe you can change the boolean variable name to make it more obvious it is a boolean